<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormNavigator</class>
 <widget class="QWidget" name="FormNavigator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>208</width>
    <height>189</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="NavigatorView" name="m_pViewNavigator" native="true">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>66</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="aLib::aWin::aToolButton" name="m_pZoomFit">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../Resource/Resource_Qt/photoLab.qrc">
         <normaloff>:/PhotoLab/magnifierFit.png</normaloff>:/PhotoLab/magnifierFit.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="aLib::aWin::aToolButton" name="m_pZoom100pcnt">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../Resource/Resource_Qt/photoLab.qrc">
         <normaloff>:/PhotoLab/magnifier1.png</normaloff>:/PhotoLab/magnifier1.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="aLib::aWin::aToolButton" name="m_pZoom200pcnt">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../Resource/Resource_Qt/photoLab.qrc">
         <normaloff>:/PhotoLab/magnifier2.png</normaloff>:/PhotoLab/magnifier2.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="m_pZoomSliderMinIcon">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="../Resource/Resource_Qt/photoLab.qrc">:/PhotoLab/size_small.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="m_pZoomSlider">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="m_pZoomSliderMaxIcon">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="../Resource/Resource_Qt/photoLab.qrc">:/PhotoLab/size_big.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>NavigatorView</class>
   <extends>QWidget</extends>
   <header>navigatorView.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>aLib::aWin::aToolButton</class>
   <extends>QToolButton</extends>
   <header location="global">aToolButton.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../Resource/Resource_Qt/photoLab.qrc"/>
 </resources>
 <connections/>
</ui>
